tests/network/sriov.go

```diff
@@ -806,3 +809,46 @@ func trimRawString2JSON(input string) string {
 	}
 	return input[startIdx : endIdx+1]
 }
+
+func getRunningVMIDomainSpec(vmi *v1.VirtualMachineInstance) (*launcherApi.DomainSpec, error) {
+	runningVMISpec := launcherApi.DomainSpec{}
+	cli := kubevirt.Client()
+
+	domXML, err := getRunningVirtualMachineInstanceDomainXML(cli, vmi)
+	if err != nil {
+		return nil, err
+	}
+
+	err = xml.Unmarshal([]byte(domXML), &runningVMISpec)
+	return &runningVMISpec, err
+}
+
+func getRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (string, error) {
+	// get current vmi
+	freshVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+	if err != nil {
+		return "", fmt.Errorf("failed to get vmi, %s", err)
+	}
+
+	vmiPod, err := libpod.GetPodByVirtualMachineInstance(freshVMI, freshVMI.Namespace)
+	if err != nil {
+		return "", err
+	}
+
+	command := []string{"virsh"}
+	if kutil.IsNonRootVMI(freshVMI) {
+		command = append(command, "-c")
+		command = append(command, "qemu+unix:///session?socket=/var/run/libvirt/virtqemud-sock")
+	}
+	command = append(command, []string{"dumpxml", vmi.Namespace + "_" + vmi.Name}...)
```

Comment:
nit:
I think virsh accept the domain ID for dumping the xml, calling `..dumpxaml 1` should work. 
It will always be ID 1 because there will always be single domain on the pod.
One could say this is not future proof in case the pod will run more then one domain, I think its not likely to happen, if it will we would probably need to revisit all tests.
Your call, feel free to ignore this common.